.github/workflows/deploy-hubs.yaml

```diff
@@ -27,6 +27,8 @@ jobs:
           - 2i2c
           - cloudbank
           - carbonplan
+          - farallon
+          # - openscapes
```

Comment:
Maybe add a comment here about when this should be un-commented (if the plan is not to do it as part of this PR)?

Reply:
I think the plan is actually to enable it on this PR as well :wink: